editor/components/Clipboard/index.tsx

```diff
@@ -198,5 +214,68 @@ export default function Clipboard() {
     }
   });
 
+  useHotkeys(
+    "left",
+    throttle(() => {
+      timeShift(-100);
+    }, THROTTLE)
+  );
+  useHotkeys(
+    "right",
+    throttle(() => {
+      // time increase 100ms
+      timeShift(100);
+    }, THROTTLE)
+  );
+
+  useHotkeys(
+    "shift+left",
+    throttle(() => {
+      // time decrease 500ms
+      timeShift(-500);
+    }, THROTTLE)
+  );
+
+  useHotkeys(
+    "shift+right",
+    throttle(() => {
+      // time increase 500ms
+      timeShift(500);
+    }, THROTTLE)
+  );
+
+  useHotkeys(
+    "down",
+    throttle(() => {
+      const editor = reactiveState.editor();
+      const currentControlIndex = reactiveState.currentControlIndex();
+      const currentPosIndex = reactiveState.currentPosIndex();
+      if (editor === CONTROL_EDITOR) {
+        setCurrentControlIndex({
+          payload: currentControlIndex + 1,
+        });
+      } else
+        setCurrentPosIndex({
+          payload: currentPosIndex + 1,
+        });
+    }, THROTTLE)
+  );
+  useHotkeys(
+    "up",
+    throttle(() => {
+      const editor = reactiveState.editor();
+      const currentControlIndex = reactiveState.currentControlIndex();
+      const currentPosIndex = reactiveState.currentPosIndex();
+      if (editor === CONTROL_EDITOR) {
+        setCurrentControlIndex({
+          payload: currentControlIndex - 1,
+        });
+      } else
+        setCurrentPosIndex({
+          payload: currentPosIndex - 1,
+        });
+    }, THROTTLE)
+  );
+
```

Comment:
Please move these to ControlBar/TimeController, it makes more sense there. Clipboard should only handle things related to Clipboard